README.md

```diff
@@ -40,6 +40,8 @@ Both JSON and YAML files can be validated against a JSON schema. The library [fg
               <!-- Duplicate keys detection is turned on by default. -->
               <detectDuplicateKeys>false</detectDuplicateKeys>
               <allowEmptyFiles>false</allowEmptyFiles>
+              <!-- Allow comments in json files is turned on by default -->
+              <allowComments>false</allowComments>
```

Comment:
should be renamed to `allowJsonComments` since comments are already allowed in yaml